docs/pages/toolchain/slot/deployment-commands/logs.md

```diff
@@ -0,0 +1,50 @@
+# logs
+
+The **`logs`** command is used to fetch and display logs from a specified deployment service within the Slot environment. This command supports real-time log streaming as well as querying historical logs, making it an essential tool for monitoring and debugging deployment activities.
+
+### Usage
+
+```sh
+slot deployments logs <Project Name> <Service> [options]
+```
+
+
+### Examples
+
+1. **Fetch the last 50 logs from a Katana service:**
+    
+    ```sh
+    slot deployments logs 'MyProject' Katana --limit 50
+    ```
+    
+2. **Stream logs from a Torii service:**
+    
+    ```sh
+    slot deployments logs 'MyProject' Torii --follow
+    ```
+    
+
+### Response and Output
```

Comment:
Those sections with response and output are great. Could we generalize it to the other commands showing real output + comments on it when required?